frontend/testing/playwright/helpers/BasePage.ts

```diff
@@ -63,4 +79,18 @@ export class BasePage extends RouterRoute {
   protected getOptionByName(name: string): Locator {
     return this.page.getByRole('option', { name });
   }
+
+  // Helper function to get a link by the text key
+  protected getLinkByTextKey(key: TextKey): Locator {
+    return this.page.getByRole('link', { name: this.textMock(key) });
+  }
```

Comment:
See that i might feels better to make helper methods for this, but I find it harder to read the tests. I prefer that the Page classes itself is using the Playwright API instead of our wrappers. Reason for this is that when i reading `getLinkByTextKey`  im not sure how it implemented. 

Keep the classes using the native API as often as possible I find the tests quite readable.

Reply:
I'll fix 😄